apache/superset
# fix(dashboard): Remove edit from url params when discarding changes

### SUMMARY
It removes the `edit` param from the URL when discarding the changes of a newly created dashboard in order to exit the edit mode.

Fixes #15673 

### BEFORE

https://user-images.githubusercontent.com/60598000/125814297-5638f4ee-b2a7-4550-8c93-b8652bb2be9d.mov

### AFTER

https://user-images.githubusercontent.com/60598000/125814640-e1812885-3988-4ff2-b720-854dcea460af.mp4

### TESTING INSTRUCTIONS
1. Click on the global + button and create a new dashboard or click on the + dashboard from dashboard page list
2. Do not add anything to the new dashboard and click on 'DISCARD CHANGE' in dashboard edit mode
3. Observe the behavior

### ADDITIONAL INFORMATION


- [x] Has associated issue: #15673
